### DISCUSSION TITLE
Add Street Complete like system

### DISCUSSION THREAD

**Discussion by TheAdventurer64** [organicmaps/organicmaps#3053] [Ideas]
So, after doing some brainstorming about ways to improve Organic Maps' editing system, I had an idea: What if we incorporated a questionare system similar to StreetComplete for POIs? 

The way it would work is by asking questions to add tags that aren't available in the normal editor, for example:

Is takeaway available? Yes/No/I Don't Know
(`takeaway=yes/no/*)`

Is delivery available? Yes/No/Don't Know
(delivery=yes/no/*)
More examples can be provided later.

To access this screen, I propose two solutions:

* Users will be able to answer questions about a place through viewing the POI info, and having a GUI asking people to answer questions about the place.

* Through the Profile screen, there can be a tab called Contribute. Through there, a list of places in the area can be shown on the map, and users can click on them to answer questions about the place.

**Comment by biodranik** [comment]
Editor functionality definitely should be improved. But I would focus on the main use case: offline editing, where others (including Street Complete) do not work. Or we can even explicitly recommend Street Complete for those who wants to edit everything around )

**Reply by biodranik** [reply]
The focus on offline editing implies that all editable info should be visible in the place info page. So if it's missing, users would want to fill it with info.

**Reply by TheAdventurer64** [reply]
> Editor functionality definitely should be improved. But I would focus on the main use case: offline editing, where others (including Street Complete) do not work. Or we can even explicitly recommend Street Complete for those who wants to edit everything around )

To my knowledge, doesn't StreetComplete work offline?

**Comment by TheAdventurer64** [comment]
#2254
